src/setup.rs

```diff
@@ -45,6 +61,38 @@ pub(crate) fn setup() {
         bb::set(&rcc.ahb1rstr, ETH_MAC_BIT);
         bb::clear(&rcc.ahb1rstr, ETH_MAC_BIT);
     }
+    #[cfg(feature = "stm32f7xx-hal")]
+    //stm32f7xx-hal does not currently have bitbanding
```

Comment:
I don't think that the F7s support bit-banding. Maybe we should document that a critical section will be created or request for one explicitly as a parameter (only for F7).

Reply:
I added a line to the comments on new in Eth. I prefer to keep the interface the same for both F4 and F7 so we don't need to duplicate example code.

Reply:
Yeah, it seems to be the most reasonable option, thanks.